integreat_cms/cms/views/push_notifications/push_notification_form_view.py

```diff
@@ -323,6 +324,23 @@ def post(self, request, *args, **kwargs):
                         pn_form.instance.sent_date = datetime.now()
                         pn_form.instance.draft = False
                         pn_form.instance.save()
```

Comment:
Do we want to update this regardless of whether it was a template or not or only when it was a "normal" push notification?

Reply:
We changed that